examples/primitive_shapes/main.cpp

```diff
@@ -65,10 +65,9 @@ int main()
 
         rinvid::RinvidGfx::clear_screen(0.2F, 0.4F, 0.4F, 1.0F);
 
-        vertical_delta = 0.0F;
+        vertical_delta   = 0.0F;
```

Comment:
This seems to be unrelated to changing color. Did you forget to run clang-format before committing refactoring changes?